packages/design-tokens/css/heart-theme.css

```diff
@@ -145,20 +438,20 @@
   --kz-var-color-peach-600-rgb-params: 183, 67, 2;
   --kz-var-color-peach-700: #903c00;
   --kz-var-color-peach-700-rgb-params: 144, 60, 0;
+  --kz-var-color-white: #ffffff;
+  --kz-var-color-white-rgb-params: 255, 255, 255;
   --kz-var-color-ash: #e6e5e5;
   --kz-var-color-ash-rgb-params: 230, 229, 229;
   --kz-var-color-stone: #f9f9f9;
   --kz-var-color-stone-rgb-params: 249, 249, 249;
-  --kz-var-color-white: #ffffff;
-  --kz-var-color-white-rgb-params: 255, 255, 255;
   --kz-var-color-iron: #8c8c97;
   --kz-var-color-iron-rgb-params: 140, 140, 151;
   --kz-var-color-slate: #524e56;
   --kz-var-color-slate-rgb-params: 82, 78, 86;
-  --kz-var-data-viz-favorable: #a5e2d2;
-  --kz-var-data-viz-favorable-rgb-params: 165, 226, 210;
-  --kz-var-data-viz-unfavorable: #f7acb9;
-  --kz-var-data-viz-unfavorable-rgb-params: 247, 172, 185;
+  --kz-var-data-viz-favorable: #7dd5bd;
```

Comment:
Double this change with Max and Marc - it seems like it's not consistent with what is in the UI Kit